client/login/magic-login/handle-emailed-link-form.jsx

```diff
@@ -254,6 +255,11 @@ class HandleEmailedLinkForm extends Component {
 			);
 		}
 
+		// transition is a GET parameter for when the user is transitioning from email user to WPCom user
+		if ( isFetching || transition ) {
```

Comment:
Will this not show the logo too when it's fetching and transition isn't set?

Reply:
Yes, and I think that's cool too: no white screen for anyone?